packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -2,13 +2,15 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
+import {EventEmitter} from 'events';
+
 import {WebexPlugin} from '@webex/webex-core';
 import {proxyEvents, tap, transferEvents} from '@webex/common';
-import {EventEmitter} from 'events';
 import jose from 'node-jose';
 import SCR from 'node-scr';
-import ensureBuffer from './ensure-buffer';
+import isUndefined from 'lodash/isUndefined';
```

Comment:
Not sure if this is needed, there are ways to determine if an object or value is undefined without using lodash, such as `(!value)` or `(typeof value === 'undefined')`, as well as many other options. Without JSDoc, it cannot be determined which approach is best.